scripts/memes.coffee

```diff
@@ -20,6 +20,10 @@ module.exports = (robot) ->
   robot.hear /\bbaber'?s\s?way\b/i, (msg) ->
     msg.send 'http://i.imgur.com/rIC8Vuc.gif'
 
+  robot.hear /bee+s?\b/i, (message) ->
+    if Math.random() <= 0.05
```

Comment:
ONLY 5% 💢 💢 :rage1:

Reply:
Make it 25% and I'd be down

Reply:
Drive-by: have you considered making this 1.0?

Reply:
:clap:

Reply:
I'm down with the 5%. Add some excitement!